Project no longer building with Flutter 3.0.0+

**Describe the bug**
The project does not build on Android (and maybe on iOS too) with Flutter 3.0.0 and above since the addition of home widgets.

**To Reproduce**
Steps to reproduce the behavior:
1. Make sure you're on the master branch of the repository and using Flutter 3.0.0 or above
2. `flutter build apk` 

**Expected behavior**
The project should build.

**Additional context**

<details>  
<summary>Error thrown while building (with verbose)</summary>

```
[   +1 ms] > Task :flutter_config:compileReleaseSources
[        ] > Task :home_widget:compileReleaseKotlin FAILED
[   +1 ms] e:
D:\Flutter\flutter_windows_1.22.6-stable\flutter\.pub-cache\hosted\pub.dartlang.org\home_widget-0.1.5\android\src\main\kotlin\es\antonborri\home_widget\HomeWidgetPlugin.kt: (19, 1):
Class 'HomeWidgetPlugin' is not abstract and does not implement abstract member public abstract fun onNewIntent(p0: Intent): Boolean defined in
io.flutter.plugin.common.PluginRegistry.NewIntentListener
[   +4 ms] e:
D:\Flutter\flutter_windows_1.22.6-stable\flutter\.pub-cache\hosted\pub.dartlang.org\home_widget-0.1.5\android\src\main\kotlin\es\antonborri\home_widget\HomeWidgetPlugin.kt: (193, 5):
'onNewIntent' overrides nothing
[+2995 ms] > Task :device_info_plus:verifyReleaseResources
[   +1 ms] > Task :connectivity_plus:verifyReleaseResources
[        ] > Task :firebase_analytics:verifyReleaseResources
[ +386 ms] > Task :firebase_crashlytics:mergeReleaseResources
[ +500 ms] > Task :firebase_core:mergeReleaseResources
[   +1 ms] > Task :flutter_config:mergeReleaseResources
[+3402 ms] > Task :app:compileFlutterBuildRelease
[  +93 ms] FAILURE: Build failed with an exception.
[   +2 ms] * What went wrong:
[   +1 ms] Execution failed for task ':home_widget:compileReleaseKotlin'.
[   +1 ms] > A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
[   +1 ms]    > Compilation error. See log for more details
[   +1 ms] * Try:
[   +1 ms] Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
[   +1 ms] * Get more help at https://help.gradle.org
[   +1 ms] BUILD FAILED in 21s
```
</details>

It seems to be an issue with the home_widget package that has been fixed in version 0.1.6 of (see https://github.com/ABausG/home_widget/issues/83). Using the latest version should work fine (it did work for me).

--- a/lib/core/constants/widget_helper.dart
+++ b/lib/core/constants/widget_helper.dart
@@ -19,6 +19,9 @@ extension WidgetTypeExtension on WidgetType {
   ///
   /// This MUST be equal to the classname of the Android WidgetProvider (see
   /// home_widget's README for more info)
+  ///
+  /// Another option would be to use qualifiedAndroidName instead of androidName
+  /// (see home_widget's updateWidget method for more info)
   String get androidName {
     switch (this) {
       case WidgetType.progress:

--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -5,7 +5,7 @@ description: The 4th generation of ÉTSMobile, the main gateway between the Éco
 # pub.dev using `pub publish`. This is preferred for private packages.
 publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 
-version: 4.11.1+1
+version: 4.11.2+1
 
 environment:
   sdk: ">=2.10.0 <3.0.0"

--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -65,7 +65,7 @@ dependencies:
   in_app_review: ^2.0.3
   device_info_plus: ^3.1.0
   pub_semver: ^2.1.0
-  home_widget: ^0.1.3
+  home_widget: ^0.1.6
 
 dev_dependencies:
   flutter_test:

--- a/test/mock/services/home_widget_mock.dart
+++ b/test/mock/services/home_widget_mock.dart
@@ -65,6 +65,7 @@ class HomeWidgetMock extends Mock {
                 'name': name,
                 'android': androidName,
                 'ios': iOSName,
+                'qualifiedAndroidName': null
               }.toString()) {
         return true;
       }

